crates/prism-recipe/harness/main.py

```
@@ -156,19 +156,17 @@ def _eval_battery_status():
 
 
 def _detect_flow():
-    """v1 (legacy single invocation) vs v3 (two-phase train/eval).
+    """v1 (legacy single invocation) vs v3 (two-phase train/eval + G1–G8).
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)